content/developer/reference/backend/testing.rst

```diff
@@ -751,6 +751,27 @@ run inside it.
   - only works locally
   - only works if the test / tour can run correctly locally
 
+``debug=True``
+**************
+
+When running a tour locally via the test suite, the ``debug=True``
+parameter can be added to the ``browser_js`` or ``start_tour``
+call::
+
+    self.start_tour("/web", code, debug=True)
+
+This will automatically open a fullscreen Chrome window with opened
+devtools and a debugger breakpoint set at the start of the tour. The tour
+is ran with the debug=assets query parameter.
+When an error is thrown, the debugger stops on the exception.
```

Comment:
If you intended to make this line a separate paragraph, you must add a second line break. If not, remove the first line break.
```suggestion

When an error is thrown, the debugger stops on the exception.
```